build.gradle

```diff
@@ -126,6 +119,8 @@ configurations {
 }
 
 dependencies {
+
+    runtimeOnly gradleApi()
```

Comment:
Why would we require the Gradle API at runtime?

Reply:
:man_facepalming: Forgot to remove after some unrelated debugging and accidentally committed. Fixed.